modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/toolbarbuttons/plugin.js

```diff
@@ -205,6 +205,71 @@
 					title: editor.lang.link.title,
 				});
 
+				/**
+				 * Spacing options are hardcoded to 1.0x, 1.5x, 2.0x, 3.0x, 4.0x and 5.0x,
+				 * so that an empty class, “mt-1 mb-1” class, up to “mt-5 mb-5” are applied respectively.
```

Comment:
My first solution is to remove it if possible 😄 .

If we really really need it, I would go with something like `.mt-h2` where`h` stands for "half", following the path Bootstrap created for the [negatives values](https://getbootstrap.com/docs/5.0/utilities/spacing/#negative-margin) `mt-n2`

/cc @pat270 @marcoscv-work

Reply:
Hey @diegonvs 

> Spacing options are hardcoded to 1.0x, 1.5x, 2.0x, 3.0x, 4.0x and 5.0x
> and also suggests using mb-1 mb-1 to mb-5 mb-5 but We don't have a 1.5x.

These describe how [AE currently works](https://github.com/liferay/alloy-editor/blob/90a5154dfb95aaa841d11884a35648f3f7459b80/src/components/buttons/button-spacing.jsx#L194-L265), not how BE should work. For some reason I could not investigate, that's the behavior we had in AE.

Given that the new BE tries to preserve AE features and interactions, the proposal here is to:
* **Make them configurable**: so that apps using the BE can decide which spacing options they provide, rather than being hardcoded by the editor
* **Have BE keep AE spacing options as defaults** when such a configuration is not being provided.

There is no need to invent new classes, please refer to the linked snippet for reference

Reply:
Great! Thank you Dani and Edu :)